modules/ROOT/pages/learn/icr-sgreplicate.adoc

```diff
@@ -1,22 +1,19 @@
 = Inter-Sync Gateway Replication
-:page-layout: article
-:page-partial:
-:page-status:
-:page-edition:
 :page-role: conceptual
-:description: Couchbase Sync Gateway -- Inter-Sync Gateway Replication provides resilient, secure, scalable bidirectional synchronization of data between cloud and edge data centers.
+:description: Use Couchbase Sync Gateway's Inter-Sync Gateway Replication to keep clusters in different data centers in sync.
 
 include::partial$_std-hdr-sgw.adoc[]
 
 :topic-group: {tg-rep-icr}
 :param-related: {xref-sgw-pg-config-properties} | {xref-sgw-pg-rest-api-admin} | {xref-sgw-pg-icr-running}
-:param-abstract: This content describes Couchbase Sync Gateway's Inter-Sync Gateway Replication feature (v2). It incorporates the content presented under Manage within the Nav bar to give an integrated view.
+:param-abstract: Inter-Sync Gateway Replication provides resilient, secure, scalable bidirectional synchronization of data between cloud and edge data centers. +
+It incorporates content presented under Manage>Replicate>Inter-Sync Gateway to give an integrated view.
 include::partial$block-abstract.adoc[]
 
 // Footnotes
-:fn-sgr1: footnote:sgr1[Inter-Sync Gateway Replication (v1) only]
-:fn-sgr2: footnote:sgr2[Inter-Sync Gateway Replication (v2) only]
-:fn-arch: footnote:arch[This architecture is also known as _ship-to-shore_ or _hub-and-spoke_.]
+:fn-sgr1: footnote:sgr1[SG{nbsp}Replicate only]
+:fn-sgr2: footnote:sgr2[Inter-Sync Gateway Replication only]
```

Comment:
I do not have context for this. Again, don't name this thing...refer to it as version of inter-Sync Gateway replication introduced in 2.8